### PR TITLE
Update to Java 11

### PR DESCRIPTION
**Pull request by lulf**

### CODE REVIEW

File: keycloak-plugin/sasl-plugin/pom.xml [hunk 1]
```diff
@@ -8,6 +8,9 @@
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <artifactId>sasl-plugin</artifactId>
+  <properties>
+    <maven.compiler.target>1.8.0</maven.compiler.target>
```
Comment (k-wall) [inline keycloak-plugin/sasl-plugin/pom.xml]: For consistency with the other plugins,  shouldn't we demand source compatibility too?

Comment (lulf) [reply keycloak-plugin/sasl-plugin/pom.xml]: Hmm, looks like your comment refer to an out dated diff. If you refresh the view, what you request is there: https://github.com/EnMasseProject/enmasse/pull/2319/files#diff-32b0296924c065e6dd96b5268041380aR11

**Comment by lulf** [comment]
@k-wall @kornys We need jdk 11 installed on Jenkins build agents

**Comment by k-wall** [comment]
From a dependencies point of view I see no reports of JDK 11 issues against the runtime dependencies.  There reports of Mockito/ByteBuddy problems with Java 11.  We will need to upgrade to 2.23.16 or higher https://github.com/mockito/mockito/blob/release/2.x/doc/release-notes/official.md

**Comment by kornys** [comment]
@lulf I can install jdk11 on jenkins agent base image

**Comment by lulf** [comment]
run tests

**Comment by lulf** [comment]
@enmasse-ci run tests

**Comment by kornys** [comment]
@enmasse-ci run tests

**Comment by lulf** [comment]
@enmasse-ci run tests

**Comment by lulf** [comment]
@enmasse-ci run tests

**Comment by lulf** [comment]
@enmasse-ci run tests

**Comment by lulf** [comment]
@enmasse-ci run tests

**Comment by lulf** [comment]
@enmasse-ci run tests

**Comment by lulf** [comment]
@enmasse-ci run tests

**Pull request merged**